src/Components/Analyzers/src/ComponentFacts.cs

```diff
@@ -39,7 +41,22 @@ public static bool IsParameter(ComponentSymbols symbols, IPropertySymbol propert
                 throw new ArgumentNullException(nameof(property));
             }
 
-            return property.GetAttributes().Any(a => a.AttributeClass == symbols.ParameterAttribute);
+            var propertyAttributes = property.GetAttributes();
+            var hasParameterAttribute = HasParameterAttribute(symbols, propertyAttributes);
+            return hasParameterAttribute;
```

Comment:
can simplify this down to one line, it's grown too big for it's own britches.

Reply:
nah, I prefer simpler debuggability